CoreDependencies/src/ivy.xml

```diff
@@ -47,7 +41,7 @@
         <dependency org="org.geotools" name="gt-referencing" rev="21.2"/>
         <!--<dependency org="org.geotools" name="gt-render" rev="21.2"/>-->
         <dependency org="org.geotools" name="gt-xml" rev="21.2"/>
-        <dependency org="org.geotools" name="gt-epsg-hsql" rev="21.2"/>
+        <!--<dependency org="org.geotools" name="gt-epsg-hsql" rev="21.2"/>-->
```

Comment:
this commented line can be removed. Or add an inline comment for the reason we need to have this line here.

Reply:
@rsabhi I think the approach used with the `org.geotools` section is to list all of the libraries `org.geotools` supports and comment the libraries we are not using. Given `gt-epsg-hsql` is not used based on my investigation I have commented it but left it there so that should someone want to include it they can. Ditto for `gt-coverage` and `gt-render`.

Reply:
My recommended approach here is to remove all the commented unused dependencies.

Reply:
Understood @rsabhi and this is what I would normally follow as well. However the `org.geotools` is a bit special and I've added a comment block to explain it (see https://github.com/constellation-app/constellation/pull/122/commits/fd9b0f2c611513a53e675c04931092c88e0bcaa7).